3_Inference/Detector.py

```diff
@@ -293,5 +303,13 @@ def get_parent_dir(n=1):
                 len(input_video_paths), end - start
             )
         )
+    # for Webcam
+    if webcam_active:
```

Comment:
Right now it does all other detection tasks first. Perhaps, you would want to use something like 

```
if input_video_paths and not webcam_active:
```
and

```
if input_image_paths and not webcam_active:
```
above?

Reply:
I have used `if input_video_paths and webcam_active==False` hope its fine. I thought this is more readable so

Reply:
to stay consistent with the rest of the code using `not` would be better.